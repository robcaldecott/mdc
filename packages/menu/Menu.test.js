import React from "react";
import { Menu, MenuItem, MenuDivider } from ".";
import renderer from "react-test-renderer";

it("renders", () => {
  expect(
    renderer.create(
      <Menu open>
        <MenuItem>Item 1</MenuItem>
        <MenuItem>Item 2</MenuItem>
        <MenuDivider />
        <MenuItem disabled>Item 3</MenuItem>
        <MenuItem>Item 4</MenuItem>
      </Menu>
    )
  ).toMatchSnapshot();
});
