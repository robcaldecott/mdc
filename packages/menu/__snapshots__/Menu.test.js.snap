// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="mdc-simple-menu mdc-simple-menu--open"
  onCancel={[Function]}
  tabIndex="-1"
>
  <ul
    aria-hidden="true"
    className="mdc-menu__items mdc-list"
    role="menu"
  >
    <li
      className="mdc-list-item"
      onClick={[Function]}
      role="menuitem"
      tabIndex="1"
    >
      Item 1
    </li>
    <li
      className="mdc-list-item"
      onClick={[Function]}
      role="menuitem"
      tabIndex="1"
    >
      Item 2
    </li>
    <li
      className="mdc-list-divider"
      role="separator"
    />
    <li
      aria-disabled="true"
      className="mdc-list-item"
      onClick={[Function]}
      role="menuitem"
      tabIndex="-1"
    >
      Item 3
    </li>
    <li
      className="mdc-list-item"
      onClick={[Function]}
      role="menuitem"
      tabIndex="1"
    >
      Item 4
    </li>
  </ul>
</div>
`;
